dev-libs/keybinder/keybinder-0.3.2-r300.ebuild

```diff
@@ -18,8 +18,8 @@ RDEPEND="x11-libs/gtk+:3[X]
 	x11-libs/libXext
 	x11-libs/libXrender
 	introspection? ( dev-libs/gobject-introspection )"
```

Comment:
Are you sure that gobject-introspection is a DEPEND here? Also, I'm pretty sure it should go to BDEPEND too because of `g-ir-compiler`.

Reply:
I moved it.  This was the same as what I wrote somewhere in #18815, where I just left it in `DEPEND` like most GNOME packages did, and it shouldn't make a difference in most cases since introspection can't be cross-compiled.

Reply:
Heh, and I thought their elaborate system did account for cross-compilation.